plugins/inputs/sqlserver/sqlqueriesV2.go

```diff
@@ -476,8 +476,6 @@ SET @SqlStatement = @SqlStatement + CAST(N' WHERE	(
 				''Readahead pages/sec'',
 				''Lazy writes/sec'',
 				''Checkpoint pages/sec'',
-				''Free pages'',
-				''Extension free pages'',
```

Comment:
why do we remove these two?

Reply:
These were added recently as part of PR - https://github.com/influxdata/telegraf/pull/8120, however these counters are rarely used (Denzil's comment above mention the same), hence were removed.

Reply:
@ssoroka - any thoughts on the above? if there are no concerns, could you please help merge the same? thank you!

Reply:
any chance that someone is depending on them?

Reply:
Unlikely this was supposed to be added to here instead by mistake was recently added to V2.

Reply:
@ssoroka / @sjwang90 - Any thoughts on the above? Please help merge the PR if no concerns. Thanks!

Reply:
@ssoroka your decision here. #8120 was merged on Sept. 2020 and thus was release in 1.17.

My question to @denzilribeiro is if it was intended to be added here, why do you now remove it. In general, what is the cost of keeping those two in there?

Reply:
@srebhan  - to be honest should have not been added as rarely ( < 0.5%) do SQL Server folks use it. Removal was more from perspective of unnecessary space, not opposed to leaving it but if 99.9% are not using it then why? But you guys' call

Reply:
It's possible if someone is using it this could break a chart. I'm on the fence here, but if it's all the same to everyone let's just leave the fields in and they can always drop the ones they don't want.

Reply:
@srebhan @ssoroka - Let me revert the change of removing the two counters, rest of the PR would remain as is. I'll push the change and ping back here.

Reply:
@srebhan @ssoroka - The two counters' removal has been reverted. If there are no other concerns, it'd be very helpful to get this merged. Thank you.